discord/member.py

```diff
@@ -366,7 +369,7 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         return (
             f'<Member id={self._user.id} name={self._user.name!r} discriminator={self._user.discriminator!r}'
-            f' bot={self._user.bot} nick={self.nick!r} guild={self.guild!r}>'
+            f' bot={self._user.bot} nick={self.nick!r} guild={self.guild!r} flags={self.flags!r}>'
```

Comment:
```suggestion
            f' bot={self._user.bot} nick={self.nick!r} guild={self.guild!r}>'
```

No need to add it to the repr.